.prettierignore

```diff
@@ -2,3 +2,5 @@
 /__fixtures__/
 /dist/
 /flow-typed/
+/npm-shrinkwrap.json
+/CONTRIBUTING.md
```

Comment:
The CONTRIBUTING.md file is here temporarily, this will be removed in #1559. This is to avoid conflict between those two PRs.